src/models/SinglePoll.ts

```diff
@@ -0,0 +1,66 @@
+import TwitterInfo from './TwitterInfo'
+import VotesInfo from './VotesInfo'
+
+export default class SinglePoll {
+  /**
+   * UTC midnight on day of the poll
+   *
+   * @memberof SinglePoll
+   */
+  scheduledStartDay: number
+
+  /**
+   * Describes whether the poll is yet to begin (tweet not posted), is in progress, or has closed to new votes.
+   *
+   * @memberof SinglePoll
+   */
+  votingStatus: VoteStates
+
+  /**
+   * Tweet info about a poll. `null` if not started.
+   *
+   * @memberof SinglePoll
+   */
+  twitterInfo: TwitterInfo | null
+
+  /**
+   * Voting info about a poll. `null` if not started, otherwise has length of 2: `0` being the first option, and `1` being the second.
+   *
+   * @memberof SinglePoll
+   */
+  votesInfo: VotesInfo | null
+
+  /**
+   * Creates an instance of `SinglePoll`.
+   *
+   * @memberof SinglePoll
+   */
```

Comment:
```suggestion
export default class SinglePoll {
  /**
   * UTC midnight on day of the poll
   */
  public scheduledStartDay: number

  /**
   * Describes whether the poll is yet to begin (tweet not posted), is in progress, or has closed to new votes.
   */
  public votingStatus: VoteStates

  /**
   * Tweet info about a poll. `null` if not started.
   */
  public twitterInfo: TwitterInfo | null

  /**
   * Voting info about a poll. `null` if not started, otherwise has length of 2: `0` being the first option, and `1` being the second.
   */
  public votesInfo: VotesInfo | null

  /**
   * Creates an instance of `SinglePoll`.
   */
```